frontend/src/app/App.tsx

```diff
@@ -68,9 +69,21 @@ const App: React.FC = () => {
     [buildStatuses, dashboardConfig, storageClasses],
   );
 
+  const isUnauthorized = React.useMemo(() => {
+    if (fetchConfigError?.request?.status === 403) {
+      return true;
+    }
+    return false;
+  }, [fetchConfigError]);
```

Comment:
No need to memoize this because the compute is negligible and the result is a stable boolean value.
```suggestion
  const isUnauthorized = fetchConfigError?.request?.status === 403;
```